.travis.yml

```diff
@@ -13,3 +13,4 @@ env:
   global:
   - ERMREST_URL: https://dev.isrd.isi.edu/ermrest
   - secure: pRyPd4GF9x2ywx3aWrG/cnxz2f2VMD6cvZWuS4J8JJaCOwqB19FpPEHFJRZLrTdNjl+RZdPrL2Tlkpt8lxOVZQ6BHhX+EMDXuR2tlpyhowhiSSemr/ce9cW17nhGLEwZNN99JLqSTHMIeG0px4L9M06iqbLnE9+HHppcQH8QPhUnMi21EpeCgZVNoiB0wBxfWel9boYnPmfNdNaT/kKk2+K9HkOqMrl9v2Wj0XPVLAATZfw684sUdVJfJx75Cvqwl1g73OklL5D3sroPJQX/c2P0kKm6MAnGAdsHbIsgjeBNbSi8VYJtYGOq6qE3H04dVCY2TYfyi4kwmu9MrKwQ0rSNf2q+9MoFOJ3Uc6AxoNW8s2yWZqWYTWlknYfM99xlHQRXP4E2rOc603TeRdTVxxcT/w9JRQCXX6pHlJuRxFmJ6EGbX2p2r2+UX6QjxMrFFv7PJALjOT5KAe281/VCAhedkaLtgHK4znmAsZkAaKKOa7YnaOvJFrgBaivfpRwFMJWnrIWAp8n0OXffc2BUvHZjgpNIMHvSCp1YMi6odoYabagc4oAZUVGSWBSTOA58sbIlmIzseBxFjrk9ygobzZaOpfkYig82ssnxpqF98tnmSxXVjBQJr99abC9liaK3/Fj0ANoYJBJCG0SDM/s7NOMJIZh45wmi359dOi/KxJg=
+  - secure: TBTq6fxGLwsimS+KClPFjFltRfNiNcj01ztbPgtFARlNTPWkoq43av3rCuBZLAeudiFqFxqArVNiQ+hRnL8PDqQNldAdZEkB9jqcGQKHh+S9u3sU9mGLFvXkbIwLo3NWCSYs1S48SBMtr7LZkCU/0jxgNWk/Y4tPjs7lBxcq9aLIUJPcygv6UXrVWwlRcY3J0/2ttflaim59XIXUy2Oxmn7ulXPthIpjwbNoQE0jRkoZgwEWKnLLmUT0fdb2nR75jnqg9NklcThoYWpJeyM0f/uSMQJsvltK9nPQrMAybiNrwIWB8ijYJ8FWU91ebzek8JZhsXEQIaDiO3xTPPL/9OVUZGUiWunjfRIA+3WDTekgJyd52l+3VaIUMnz/+mDiKDqPh6ena9dpaRNcdGT72CPaewmcibGB+1iDJXjm4ph5ySNncI0zhDRMBD49hhrW9C+x2Y05qb13wGAdtoLIA9R1m/CZcXuC0LrUMK2KaWEhjFJKJ51mXi0BXYG4scVePmYp2Mrs6sjdhRnceIu7qxTfdvLq8g6WDhQrpV0LsKGHR3J40ThNqA2aLzOfuCwCBGeJ0qVknkAd4H7xRT9a3OuQUUOdzPZ5ltBxQLGxk10uO8DivaOcpgkt13zf3eVZeMGz9pTyGJk9v4vGyeq8wKb2x+ZW/Bz/xVrDcci9BfM=
```

Comment:
Instead, let's please **stop** writing travis tests that talk to our servers. The travis recipe should include the ermrest install steps to deploy a transient ermrest service on the temporary travis build VM, then run tests against that instead.